apps/web/src/components/form/MultiSelect.tsx

```diff
@@ -45,13 +45,13 @@ export const selectStyleOverride: StylesConfig<OptionType, true> = {
     transform: 'scale(0.8, 0.85)',
   }),
   input: styles => ({ ...styles }),
-  control: styles => ({
+  control: (styles, { isDisabled }) => ({
     ...styles,
     display: 'flex',
-    padding: '2px',
+    padding: '1px',
     border: '0',
-    borderBottom: '1px solid rgb(49, 49, 50)',
-    background: 'rgb(243, 244, 246)',
+    borderBottom: isDisabled ? '2px solid rgb(110, 110, 110)' : '2px solid #313132',
+    background: isDisabled ? 'rgb(215, 215, 215)' : '#F5F5F5',
```

Comment:
Should we add these colours to the theme?

Reply:
Good thinking but we don't really have a global theme object like in MUI, colors are stored in tailwind/it's config. This is a one-off because this library only allows CSSOM for styling afaik. I don't think it's worth the effort of a global theme/color object just for this component

Switching to downshift would allow us to style with tailwind